### DISCUSSION TITLE
Thanks For Students Developer Pack

### DISCUSSION THREAD

**Discussion by ChanvithaPraveen** [community/community#69035] [GitHub Education] [answered]
### Hi everyone,

Today I got the Students Developer Pack. Thank you GitHub Team.

**Comment by mantasu** [comment, marked as answer]
Hey, congrats! I've actually recently posted some [primary suggestions](https://github.com/orgs/community/discussions/68098#discussioncomment-7092052) of utilizing the Dev Pack, it may be helpful for you!

**Comment by ChanvithaPraveen** [comment]
Thank you for the comment

**Comment by kanduru-abhiram** [comment]
Hey @ChanvithaPraveen, congrats for getting the dev pack, you can start utilizing the benefits, do check the below link for more info on the benefits of the student pack.

https://education.github.com/pack/offers

Please accept this as a solution if helped, thanks
